CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Main (unreleased)
 
+- [FEATURE] New integration: grok_exporter (@roobalimsab)
```

Comment:
Can you move this below the breaking change notification?

Reply:
Done